hack/Dockerfile.unit

```diff
@@ -1,6 +1,10 @@
 FROM registry.hub.docker.com/library/golang:1.13
 
+COPY . /usr/local/kubebuilder/
```

Comment:
I think you should not place this in /usr/local/kubebuilder. maybe in /CAPM3 and you delete that folder afterwards ?

Reply:
okay